apps/core/components/ItineraryCard/FlightTimes.js

```diff
@@ -35,7 +36,7 @@ const styles = StyleSheet.create({
     padding: 5,
     web: {
       fontWeight: '500',
-      color: '#2e353b', // @TODO repeating value - should be added to design-tokens
+      color: margaritaTokens.textColorBlueDark,
```

Comment:
The convention in `orbit-design-tokens` is to declare only `textColor` for some `types`, e.g. `textColorPrimary`. Also, we are using `colorText` not `textColor`, but it's totally up to you.

This doesn't apply for `palette` tokens, so there are two options/ways I suggest:
- declaring `paletteBlueDark` token and using it every needed
- declaring component specific tokens, e.g. `colorTextFlighTime` for this case

The second option is probably better because we will know much easier what differences you have and what's needs to be solved by `orbit-design-tokens`.

Reply:
Really thanks for the feedback. 
For now I just used `paletteBlueDark` because we want to mainly collect generic repeating style values for now.
But we will definitely sort them to something more specific in some next iteration. Probably like suggested `colorTextFlighTime: paletteBlueDark` as it is defined in Orbit.